jobs/Backend/Task/ExchangeRateProvider/ExchangeRate.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace ExchangeRateProvider
+{
+    public class ExchangeRate
+    {
+        public ExchangeRate(int amount, CurrencyCode sourceCurrency, CurrencyCode targetCurrency, decimal rate)
+        {
+            Amount = amount;
+            SourceCurrency = sourceCurrency;
+            TargetCurrency = targetCurrency;
+            Rate = rate;
+        }
+
+        public int Amount { get; }
```

Comment:
I have decided to alter the original class a little bit and added this integer property called Amount. Some currencies are using higher base than just one.